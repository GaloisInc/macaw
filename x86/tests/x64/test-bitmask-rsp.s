        .text
        .globl _start
_start:
        movq %rsp, %rax
        subq $64, %rsp
        andq $-32, %rsp
        movq %rax, 32(%rsp)

        movq 32(%rsp), %rsi
        leaq (%rsi), %rsp

        movq $60, %rax
        syscall
        nop
        ret
