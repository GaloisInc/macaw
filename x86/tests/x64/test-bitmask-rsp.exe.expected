R { funcs = [ (Addr 0 0x401000, [(Addr 0 0x401000,33)])
            , (Addr 0 0x401021, [(Addr 0 0x401021, 2)])
            ]
  , ignoreBlocks = []
  }
